tutorial/create-a-simple-squid.md

```diff
@@ -30,22 +30,22 @@ Next, clone the created repository (be careful of changing `<account>` with your
 git clone git@github.com:<account>/squid-template.git
 ```
 
-For reference on the complete work, you can find the entire project [here](https://github.com/RaekwonIII/squid-template/tree/crust-integration-demo).
+For reference on the complete work, you can find the entire project [here](https://github.com/RaekwonIII/squid-template/tree/crust-integration-demo) - TODO(USE A PROPER LINK).
 
 ### Run the project
 
 Next, just follow the [Quickstart](../quickstart.md) to get the project up and running, here's a list of commands to run in quick succession:
 
-```bash
-npm ci
-npm run build
-docker compose up -d
-npx sqd db create
```

Comment:
If I go through tutorial again I get the message that `Database 'squid' already exists`.